### PR TITLE
Add lookup with tag

### PR DESCRIPTION
**Pull request by florian-zeidler**
@rebuy-de/it-platform Please review, we can now use tags to lookup a service.

### CODE REVIEW

**Comment by svenwltr** [comment]
:+1:

File: src/main/java/com/rebuy/consul/ConsulService.java [hunk 1]
```diff
@@ -76,6 +86,11 @@ private void verifyServiceFound(String serviceName, List<CatalogService> service
         return client.getCatalogService(serviceName, QueryParams.DEFAULT).getValue();
     }
 
+    private List<CatalogService> fetchServices(String serviceName, String tag)
+    {
+        return client.getCatalogService(serviceName, tag, QueryParams.DEFAULT).getValue();
```
Comment (ghost) [inline src/main/java/com/rebuy/consul/ConsulService.java:91]: I don't think adding an extra function for one line of code increases readability.

Comment (florian-zeidler) [inline src/main/java/com/rebuy/consul/ConsulService.java:91]: I think it does, because we keep a single level of abstraction.

**Comment by ghost** [comment]
:+1:

**Pull request merged**